zeus/monitor/carbon.py

```diff
@@ -11,12 +11,12 @@
 
 from typing import Literal
 from datetime import datetime, timezone, timedelta
+from dateutil import parser
```

Comment:
I think this is a new dependency that has to be specified in `pyproject.toml` without pinning the version.